libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/ExitSignCreator.java

```diff
@@ -0,0 +1,67 @@
+package com.mapbox.services.android.navigation.ui.v5.instruction;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.mapbox.api.directions.v5.models.BannerComponents;
+import com.mapbox.services.android.navigation.ui.v5.R;
+
+import java.util.List;
+
+class ExitSignCreator extends NodeCreator<BannerComponentNode, ExitSignVerifier> {
+  private String exitNumber;
+  private int startIndex;
+  private TextViewUtils textViewUtils;
+  private String modifier;
+
+  ExitSignCreator() {
+    super(new ExitSignVerifier());
+    textViewUtils = new TextViewUtils();
+  }
+
+  @Override
+  BannerComponentNode setupNode(BannerComponents components, int index, int startIndex, String
+    modifier) {
+    if (components.type().equals("exit")) {
```

Comment:
Have we thought about extracting these magic numbers i.e. `"exit"`, `"exit-number"` and `"left"` into constants?